pkg/api/types.go

```diff
@@ -438,6 +442,28 @@ type GlusterfsVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// RBDVolumeSource represents a Rados Block Device Mount that lasts the lifetime of a pod
+type RBDVolumeSource struct {
+	// Required: CephMonitor is the ip:port of the ceph cluster monitor
+	CephMonitor string `json:"monitor" description:"ip:port of the ceph cluster monitor"`
```

Comment:
There are normally N monitors; is this a comma separated list, or a json list?  Alternatively ceph can also take a dns name and it'll form a list from the A or AAAA records, but most users don't have DNS set up that way for their clusters.